src/entry/Entry.tsx

```diff
@@ -102,16 +71,17 @@ const Entry: React.FC<EntryProps> = ({
       </BaseLayout>
     );
   }
+  const transformedData = uniProtKbConverter(data);
 
   const sections = UniProtKBEntryConfig.map((section) => ({
     label: section.name,
     id: section.name,
 
     disabled:
       section.name === EntrySection.ExternalLinks
-        ? !hasExternalLinks(entryData)
+        ? !hasExternalLinks(transformedData)
         : // eslint-disable-next-line @typescript-eslint/no-explicit-any
-          !hasContent((entryData as any)[section.name]),
+          !hasContent((transformedData as any)[section.name]),
```

Comment:
`as any` (and the eslint exception) not needed anymore. Maybe eslint got smarter with the latest package upgrades.